conduit/plugins/processors/filterprocessor/sample.yaml

```diff
@@ -2,8 +2,12 @@ name: filter_processor
 config:
   # Filters is a list of boolean expressions that can search the payset transactions.
   # See README for more info.
```

Comment:
This one needs to go, it's the filters doc

Reply:
removed.